Sources/SwiftSignal/Signal.swift

```diff
@@ -10,16 +10,26 @@
 /// Signals track a single value that changes over time. Signals are the
 /// source of truth for other reactive primitives, and have no external
 /// dependencies.
+@propertyWrapper
 public class Signal<T: Equatable>: Source {
     
     private(set) var value: T
     private(set) var observers = Set<AnyObserver>()
     
+    public var wrappedValue: T {
+        get { self() }
+        set { write(newValue) }
+    }
+    
     /// Creates a signal with an initial value.
     public init(initialValue: T) {
         self.value = initialValue
     }
     
+    public init(wrappedValue: T) {
```

Comment:
Nit: this should be a convenience initializer because the existing designated initializer is sufficient.

Reply:
Thanks!